Resources/Public/js/VersionSevenPointTwo/DragDrop.js

```diff
@@ -0,0 +1,159 @@
+/*
+ * This file is part of the FluidTYPO3/Flux project under GPLv2 or later.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.md file that was distributed with this source code.
+ */
+
+/**
+ * Important! The original DragDrop.js must be replaced with this one.
+ * so we undefine the module via requirejs!
+ */
+requirejs.undef("TYPO3/CMS/Backend/LayoutModule/DragDrop");
```

Comment:
Is there no way to avoid this? Personally I don't consider it safe to replace core JavaScript. What is it precisely that we need these modified methods to do - would it be possible to contribute this to the core itself, as simple flexibility/compatibility precautions that increase support for modules/hooks that produce HTML markup in the page module.